src/main/java/com/ibm/cloud/cloudant/kafka/tasks/SourceChangesTask.java

```diff
@@ -86,9 +88,7 @@ public List<SourceRecord> poll() throws InterruptedException {
                         }
                     })).collect(Collectors.toList());
 
-            LOG.info("Return " + records.size() / topics.size() + " records with last offset "
-                    + latestSequenceNumber);
-
+            LOG.debug("Return " + records.size() + " records with last offset " + latestSequenceNumber);
```

Comment:
Given this is going to be output signitifcantly less frequently following the level change I think it makes sense to use the parameterized form and avoid the costs of concatenation and creation of strings that won't be used most of the time.

```suggestion
            LOG.debug("Return {} records with last offset {}", records.size(), latestSequenceNumber);
```

We should probably make that kind of change throughout (at least for logging levels that are likely disabled) but obviously that is out of scope for this PR.